lib/resty/waf.lua

```diff
@@ -165,7 +165,7 @@ local function _do_transform(self, collection, transform)
 				return collection -- dont transform if the collection was nil, i.e. a specific arg key dne
 			end
 
-			logger.log(self, "doing transform of type " .. transform .. " on collection value " .. tostring(collection))
+			--_LOG_"doing transform of type " .. transform .. " on collection value " .. tostring(collection)
```

Comment:
We need acceptance tests to look for what we expect debug log entries to look like